docs/nidcpower/rep_caps.rst

```diff
@@ -42,9 +56,18 @@ instruments
 
         .. code:: python
 
-            session.instruments['0-2'].channel_enabled = True
+            session.instruments[0].channel_enabled = True
+
+        sets :py:attr:`channel_enabled` to :python:`True` for instruments 0.
```

Comment:
If we're going to show an example, we should make the documentation smarter.
How?

a) use a property that actually supports the repeated capability, and use that in the example. This can be done by us telling the generator what to use via metadata or by searching for a property that supports it. Probably easier to explicitly add to metadata.
b) pass in `"Dev1"` instead of `0`. This is not something we can do without additional metadata though.

Then we can remove the silly "Note that our codegenerator isn't smart enough to do this well"

Reply:
To better illustrate what I'm saying, we'd need additional Python-specific metadata, like:

```
    'repeated_capabilities': [
        {
            'prefix': '',
            'python_name': 'channels',
            'property_for_docs':'channel enabled',
            'value_for_docs':'0',
        },
```

Reply:
I think I may have time on Monday to experiment with updating the metadata to achieve this.

>b) pass in "Dev1" instead of 0. This is not something we can do without additional metadata though.

There's a repeated capability that won't accept [0]? I'll need to change my edits higher up for recommended types to pass.

Reply:
yes, for example the `instruments` repeated capability takes in instrument names (string)

Reply:
> use a property that actually supports the repeated capability, and use that in the example. 


Please check that there is at least one property for every supported repeated capability type. Else, you'll have plumb the logic for using methods.